pages/index.js

```diff
@@ -1,30 +1,35 @@
-import { useState, useEffect } from 'react'
-import FrontPageLayout from '@/components/layouts/FrontPageLayout';
+import { useState, useEffect } from "react";
+import FrontPageLayout from "@/components/layouts/FrontPageLayout";
 import { withIronSessionSsr } from "iron-session/next";
-import { sessionOptions } from '@/lib/Session';
+import { sessionOptions } from "@/lib/Session";
+import { slsFetch } from '@/components/Util'; 
 
-export default function Index() {
-  return (
-    <FrontPageLayout></FrontPageLayout>
-  );
-  }
+export default function Index(props) {
+  return <FrontPageLayout {...props} ></FrontPageLayout>;
+}
 
 export const getServerSideProps = withIronSessionSsr(
   async function getServerSideProps({ req, res }) {
-  
-  let homepage;
-  if(homepage = req.session?.cache?.homepage) {
-    return {
-       redirect: {
-        permanent: false,
-        destination: `/${homepage}`,
-      }     
-    }
-  }
+   try{
+    const response = await slsFetch(`http://localhost:3001/api/resources/mainlogo`);
```

Comment:
We have hardcoded `localhost:3001`. This will work only on localhost, and will break on staging and production.
```suggestion
    const response = await slsFetch(`/api/resources/mainlogo`);
```

Reply:
Oh. I didnt notice this.😅 I think I just copied the url from weburl and pasted it here when doing some testing and forgot to remove the localhost. 😅 anyway thanks for the correction sir.❤️❤️❤️